.github/workflows/sdk-python-ci.yml

```
@@ -40,3 +38,24 @@
           GROQ_API_KEY: ${{ secrets.PYTHON_SDK_GROQ_API_KEY }}
           OPENAI_API_KEY: ${{ secrets.PYTHON_SDK_OPENAI_API_KEY }}
         run: uv run pytest
+
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)